.yarnrc.yml

```diff
@@ -0,0 +1,4 @@
+compressionLevel: mixed
+enableGlobalCache: false
+nodeLinker: node-modules
+npmRegistryServer: "https://registry.yarnpkg.com"
```

Comment:
This actually works and doesn't default to the Symphony private registry if configured.